packages/starknet-snap/src/utils/starknetUtils.ts

```diff
@@ -186,6 +186,21 @@ export const declareContract = async (
   });
 };
 
+export const getAccountNonce = async (
+  network: Network,
+  senderAddress: string,
+  privateKey: string | Uint8Array,
+  cairoVersion?: CairoVersion,
+): Promise<BigNumberish> => {
```

Comment:
we try not to depends on their type too much, 
as your return type is int anyway, it wont become bigint
but im okay to keep BigNumberish in this PR,